test/dotnet-add-p2p.Tests/dotnet-add-p2p.Tests.csproj

```diff
@@ -20,6 +20,12 @@
     <ProjectReference Include="..\..\src\Microsoft.DotNet.Cli.Utils\Microsoft.DotNet.Cli.Utils.csproj">
       <FromP2P>true</FromP2P>
     </ProjectReference>
+    <ProjectReference Include="..\..\src\Microsoft.DotNet.Cli.Sln.Internal\Microsoft.DotNet.Cli.Sln.Internal.csproj">
+      <FromP2P>true</FromP2P>
```

Comment:
That's a migration artifact, so that people could then clean up their csproj files once transitive dependency was supported in the SDK. Which it is now. But in any way, you should not add that metadata by hand.